app/classifier/index.cjsx

```diff
@@ -342,7 +301,6 @@ Classifier = React.createClass
       </nav>
     </div>
 
-
   renderGravitySpyGoldStandard: (classification) ->
```

Comment:
This method is also a kind of summary that could be moved into the `ClassificationSummary` component. There's a method on L378 that checks if this summary should be shown.